subprojects/common/analysis/src/main/java/hu/bme/mit/theta/analysis/algorithm/ArgEdge.java

```diff
@@ -18,36 +18,10 @@
 import hu.bme.mit.theta.analysis.Action;
 import hu.bme.mit.theta.analysis.State;
 
-import java.util.Objects;
-
 public final class ArgEdge<S extends State, A extends Action> {
     private static final int HASH_SEED = 3653;
```

Comment:
Please remove unused HASH_SEED and hashCode fields from here (and from other modified classes as well).

Reply:
Now, that I am looking at it, it seems that the states attribute of ArgTrace is also unused (it is set but not used):
[states attribute](https://github.com/ftsrg/theta/blob/780fa90aba4a2b56d74ea08342ad00b87b107347/subprojects/common/analysis/src/main/java/hu/bme/mit/theta/analysis/algorithm/ArgTrace.java#L42C37-L42C43) (I cannot add a review comment there because this line has not changed...)

Reply:
Done, thanks!